integrationtest/TargetProjects/TargetProject/TargetProject.csproj

```diff
@@ -1,9 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netstandard2.0</TargetFrameworks>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <LangVersion>7.3</LangVersion>
+    <NullableContextOptions>enable</NullableContextOptions>
```

Comment:
NullableContextOptions is the old property name and has been replaced by Nullable